examples/ReduxExample/package.json

```diff
@@ -22,17 +22,17 @@
     ]
   },
   "dependencies": {
-    "expo": "^21.0.1",
+    "expo": "^22.0.0",
     "prop-types": "^15.5.10",
-    "react": "16.0.0-alpha.12",
-    "react-native": "~0.48.4",
+    "react": "16.0.0-beta.5",
+    "react-native": "https://github.com/expo/react-native/archive/sdk-22.0.1.tar.gz",
```

Comment:
I don't think we should depend on the Expo fork here, just vanilla react-native

Reply:
Yeah sorry, I ended up changing it to `^0.49.3` in a later commit

Reply:
Looks like it was changed in the `NavigationPlayground` example but not this one.

Reply:
You're totally right, my bad